homeassistant/components/number/const.py

```diff
@@ -165,6 +166,15 @@ class NumberDeviceClass(StrEnum):
     Unit of measurement: `Wh`, `kWh`, `MWh`, `MJ`, `GJ`
     """
 
+    ENERGY_DISTANCE = "energy_distance"
```

Comment:
This PR is changing our entity model. Before we can review this PR there needs to be approval in a discussion in our architecture repository.

https://github.com/home-assistant/architecture/discussions

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model

Reply:
Awesome! While we await the result of that discussion, I'll go ahead and close up this PR for now.

We can re-open it when the proposal is accepted.

../Frenck